website/docs/r/workspace.html.markdown

```diff
@@ -12,6 +12,8 @@ Provides a workspace resource.
 
 ~> **NOTE:** Using `global_remote_state` or `remote_state_consumer_ids` requires using the provider with Terraform Cloud or an instance of Terraform Enterprise at least as recent as v202104-1.
 
+~> **NOTE:** Using `trigger-patterns` requires using the provider with Terraform Cloud.
```

Comment:
I wonder if we could put this under the actual trigger patterns bullet? I also wonder if it needs to be a note, or if we could incorporate it into the bullet text. The reason why is that I'm thinking it'd be easy for someone to kind of skip right to the available attributes and miss this note. I think it'd be more helpful for users to see the information about its requirements in context:

"`trigger_prefixes` - (Optional) List of prefixes that describe the paths Terraform Cloud monitors for changes, in addition to the working directory. Trigger prefixes are always appended to the root directory of the repository. Terraform Cloud or Terraform Enterprise will start a run when files are changed in any directory path matching the provided set of prefixes. Mutually exclusive with `trigger-patterns`. Only available for Terraform Cloud."

Reply:
Good point -> 8a22a371f415a88f25821d04a4ab11741b3ee445